src/univariate/continuous/skewexponentialpower.jl

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
If you use
```suggestion
@doc raw"""
```
you can use single backslashes, e.g. ``\mu`` instead of ``\\mu`` (you don't have to change it).